### DISCUSSION TITLE
synology container manager pull a latest tagged image but version was 1.23.1

### DISCUSSION THREAD

**Discussion by lish-lib** [dani-garcia/vaultwarden#4089] [Q&A]
same as title

when i recently update my vaultwarden container by using watchtower

my vaultwarden rollback to version 1.23.1, and as the reault my bitwarden clients couldnot login

and manually pulling the latest image is also shown as builded in 2021. 

now i have pulled the 1.30.1 tag image and rebuilded

**Comment by BlackDex** [comment]
You probably use a mirror which doesn't have the correct latest tagged image pulled.
